reinforcement_learning/rlclientlib/model_mgmt/model_mgmt.cc

```diff
@@ -1,28 +1,52 @@
 #include "model_mgmt.h"
+
 #include <new>
 
 namespace reinforcement_learning {
   namespace model_management {
 
-    model_data::model_data() 
-    :_data{nullptr}, _data_sz { 0 }, _refresh_count { 0 } {}
+    model_data::model_data() :
+      _data{nullptr},
+      _data_sz { 0 },
+      _refresh_count { 0 }
+    {}
+
+    model_data::~model_data() {
+      free();
```

Comment:
>free(); [](start = 6, length = 7)

Since have a non-trivial destructor, we need to ensure this object has proper copy/assignment etc.  Previously resource was explicitly released.